Changelog.md

```diff
@@ -1,6 +1,64 @@
 ## Ignition Common 4.x
 
-## Ignition Common 4.X.X
+## Ignition Common 4.X.X (202X-XX-XX)
+
+## Ignition Common 4.1.0 (2021-06-22)
+
+1. Make KeyEvent rule-of-five compliant
+    * [Pull request #224](https://github.com/ignitionrobotics/ign-common/pull/224)
+
+1. Fix `av_*` API usage for deprecations
+    * [Pull request #220](https://github.com/ignitionrobotics/ign-common/pull/220)
+
+1. Set project-wide standard to C++17
+    * [Pull request #221](https://github.com/ignitionrobotics/ign-common/pull/221)
+
+1. Remove `CMAKE_CXX_FLAGS` from test targets
+    * [Pull request #214](https://github.com/ignitionrobotics/ign-common/pull/214)
+
+1. Support loading PBR textures in OBJLoader
+    * [Pull request #216](https://github.com/ignitionrobotics/ign-common/pull/216)
+
+1. Remove `tools/code_check` and update codecov
+    * [Pull request #219](https://github.com/ignitionrobotics/ign-common/pull/219)
+
+1. Backport collada fixes
+    * [Pull request #211](https://github.com/ignitionrobotics/ign-common/pull/211)
```

Comment:
I think this backport PR could be removed from the changelog for ignition-common4